chore(formatter): align parser options of example with prettier conformance

Dunqing opened the pull request:
Keep them as the same, so that it wouldn't confuse you.
https://github.com/oxc-project/oxc/blob/36d5a795f053662238eead81021efad4ed0bf332/tasks/prettier_conformance/src/lib.rs#L423-L429

Boshen commented:
### Merge activity

* **Jun 26, 3:11 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Dunqing committed:
chore(formatter): align parser options of example with prettier conformance (#11905)

Keep them as the same, so that it wouldn't confuse you.
https://github.com/oxc-project/oxc/blob/36d5a795f053662238eead81021efad4ed0bf332/tasks/prettier_conformance/src/lib.rs#L423-L429
